riva/clients/nmt/riva_nmt_streaming_s2s_client.cc

```diff
@@ -54,6 +54,7 @@ DEFINE_int32(num_parallel_requests, 1, "Number of parallel requests to keep in f
 DEFINE_int32(chunk_duration_ms, 100, "Chunk duration in milliseconds");
 DEFINE_string(source_language_code, "en-US", "Language code for the input speech");
 DEFINE_string(target_language_code, "en-US", "Language code for the output speech");
+DEFINE_string(dnt_phrases_file, "", "File with a list of words to do not translate. One line per word.");
```

Comment:
can we name is `dnt_words_file` since we expect user to provide words inside of phrases

Reply:
> can we name is `dnt_words_file` since we expect user to provide words instead of phrases

its done